src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -818,7 +820,12 @@ public void setValueAt(Object value, int r, int c) {
 				if (c == 3) {
 					Lot lot = invRow.getLot();
 					if(lot == null) {
-						return ;
+						Lot lotTosotore = getLot(value.toString());
```

Comment:
Fix `lotToStore`?

Reply:
yes it is to showw you the window to create lot if you cancel the action end.look lines 824, 825

Reply:
I mean the variable spelling: `lotToStore`, not lotTosotore